### PR TITLE
[Stacked plots] fixes scroll issue and removed redundant calls to backend

### LINKED ISSUE #3584: [Stacked Plots] When adding more than 10 plots, plots get squashed and users cant scroll to reach plots
When users add more than 10 plots to stacked plots, the plots get squashed and become un-usable. 

We need to add a min-height and overflow-y: scroll to allow users to use stacked plots in these cases.

### PR DESCRIPTION
**Pull request by deeptailor**
Closes #3584 

## Author Checklist
Changes address original issue? Yes
Unit tests included and/or updated with changes? N/A Legacy code
Command line build passes? Yes
Changes have been smoke-tested? Yes
Testing instructions included? Yes, in issue

### CODE REVIEW

File: src/styles/_legacy-plots.scss [hunk 1]
```diff
@@ -79,7 +80,8 @@ mct-plot {
         display: flex;
         flex: 1 1 auto;
         flex-direction: column;
-        overflow: hidden;
+        overflow-y: scroll;
```
Comment (nikhilmandlik) [inline src/styles/_legacy-plots.scss:83]: how about `auto` instead if `scroll`?

**Comment by nikhilmandlik** [comment]
one small change else LGTM.

**Review by shefalijoshi — APPROVED** [review]
LGTM

**Review by nikhilmandlik — APPROVED** [review]
LGTM

**Pull request merged**